### ISSUE TITLE
Merging of local objects based on (trying to add an existing) external data source

### ISSUE THREAD

**Issue by koeaw** [acdh-oeaw/apis-core-rdf#1036]
Follow-on from #809, specifically https://github.com/acdh-oeaw/apis-core-rdf/issues/809#issuecomment-2203245269, https://github.com/acdh-oeaw/apis-core-rdf/issues/809#issuecomment-2203855550:

Current behaviour when trying to merge external data into a local object when the external data source is already linked on another object is to merge the two local objects.

I'm not sure if this is the best course of action / if automagic merging of local objects based on external (meta) data is what users would expect to happen.

It's very easy to merge two local objects if that's the intention since those appear first in the merge field's search results. Or even when it's not the intention, i.e. when the user wasn't aware that a local duplicate already exited -> same thing, it would turn up at the top of the list, from where users could inspect it and decide to merge the two local objects if that's what should happen.

But in cases where local objects / external data sources "look" the same but are actually separate entities and should remain separate, I can see users accidentally selecting the wrong external data source to add meta information and thus inadvertently merging the two local objects. Example: identically named places which are easy to confuse, e.g. cities and federal states which share a name (Vienna, Salzburg).

TL;DR I think (destructive) actions like merging need to be really transparent to users. I'd therefore limit merging of local objects to users having to actively, specifically select local objects to merge together.

A temp. message, similar to the one we now show for object duplication, to let users know the data source they are trying to add to object X already exists (on other local object Y) would _add_ transparency. We could then add the current mechanism _to that_ if that were useful, e.g. by extending the message with a link or button that users could (would have to) click to merge the two local objects.

**Comment by b1rger** [comment]
Implemented in ba7ad4c2b450e83069b960d012ff4e3f544306e8
